champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/warlock/sword/Wreath.java

```diff
@@ -78,6 +114,40 @@ public double getSlowDuration(int level) {
         return baseSlowDuration + ((level - 1) * slowDurationIncreasePerLevel);
     }
 
+    @Override
+    public boolean canUse(Player player) {
+        if (charges.containsKey(player) && charges.get(player).getCharges() > 0) {
```

Comment:
store charges.get(player) in a variable and just check if its null, that way you dont need to do 2 checks just to get the amount of charges